README.md

```diff
@@ -127,6 +127,17 @@ To test the action locally, execute the `run:local:report` command. This reports
 
 For example: `yarn run:local:report ./test-data/old ./test-data/new`
 
+### Release process
+
+1. Merge all changes into the `main` branch.
+1. Create a new [GitHub release](https://github.com/Rebilly/lexi/releases/new):
+    1. The version number must follow [semantic versioning](https://semver.org/).
+    1. Enter the new tag in the release form, and choose "Create a new tag on publish". The tag must be prefixed with a `v`, for example `v1.0.0`.
```

Comment:
suggestion: 
I think this information is supplementary to this step, and not really a step on its own
```suggestion
    1. Enter the new tag in the release form, and choose "Create a new tag on publish". The tag must be prefixed with a `v`, for example `v1.0.0`. \
    The version number must follow [semantic versioning](https://semver.org/).
```

Reply:
Good catch!